apps/app/src/views/routes/Order.tsx

```diff
@@ -62,26 +63,58 @@ export const ORDER_FULFILLMENT_TYPE_MAP = {
   [types.FulfillmentType.MailOrder]: 'Mail order'
 };
 
-export const FILL_STATE_MAP: object = {
-  CANCELED: 'Canceled',
-  NEW: 'New',
-  SCHEDULED: 'Scheduled',
-  SENT: 'Sent'
-};
-export const FILL_COLOR_MAP: object = {
-  CANCELED: 'gray',
-  NEW: 'green',
-  SCHEDULED: 'orange',
-  SENT: 'yellow'
+const CancelOrderAlert = ({
+  orderState,
+  fulfillmentState
+}: {
+  orderState: OrderState;
+  fulfillmentState: OrderFulfillmentState;
+}) => {
+  if (orderState === 'CANCELED') {
+    return (
+      <Alert colorScheme="gray">
+        <AlertIcon />
+        This order has been canceled
+      </Alert>
+    );
+  }
+  if (orderState === 'COMPLETED') {
+    return (
+      <Alert colorScheme="gray">
+        <AlertIcon />
+        This order has been completed
+      </Alert>
+    );
+  }
+  if (fulfillmentState === 'PICKED_UP') {
+    return (
+      <Alert status="warning">
+        <AlertIcon />
+        This order has been picked up, but we can't cancel the remaining fills. Please call the
+        pharmacy.
+      </Alert>
+    );
+  }
+  if (fulfillmentState === 'RECEIVED' || fulfillmentState === 'READY') {
+    return (
+      <Alert status="warning">
+        <AlertIcon />
+        This order may have been picked up, but we can cancel the remaining fills.
```

Comment:
maybe "but **you** can cancel the remaining fills."?

Reply:
Gonna leave for now since this has gone through a few people, can revisit copy if it's a problem.